lib/Runtime/Types/SimpleDictionaryTypeHandler.cpp

```diff
@@ -103,8 +103,12 @@ namespace Js
 
     bool TPropertyKey_IsInternalPropertyId(JavascriptString* key)
     {
+#ifdef DEBUG
+        Js::PropertyRecord const * localPropertyRecord;
+        key->GetPropertyRecord(&localPropertyRecord);
         // WARNING: This will return false for PropertyStrings that are actually InternalPropertyIds
-        Assert(!PropertyString::Is(key) || !IsInternalPropertyId(((PropertyString*)key)->GetPropertyRecord()->GetPropertyId()));
```

Comment:
>!IsInternalPropertyId(((PropertyString*)key)->GetPropertyRecord()->GetPropertyId()) [](start = 43, length = 83)

nit: Since in this `||` branch key is PropertyString, can be simplified to `!IsInternalPropertyId(((PropertyString*)key)->GetPropertyId())` without adding above 2 lines.